usbsdmux/usbsdmux.py

```diff
@@ -88,48 +84,51 @@ def mode_disconnect(self, wait=True):
         wait -- Command will block for some time until the voltage-supply of
         the sd-card is known to be close to zero
         """
-        pass
+        raise NotImplementedError()
 
-    @abc.abstractmethod
     def mode_DUT(self, wait=True):
         """
         Switches the MicroSD-Card to the DUT.
 
         This Command will issue a disconnect first to make sure the SD-card
         has been properly disconnected from both sides and its supply was off.
         """
-        pass
+        raise NotImplementedError()
 
-    @abc.abstractmethod
     def mode_host(self, wait=True):
         """
         Switches the MicroSD-Card to the Host.
 
         This Command will issue a disconnect first to make sure the SD-card
         has been properly disconnected from both sides and its supply was off.
         """
-        pass
+        raise NotImplementedError()
 
-    @abc.abstractmethod
     def gpio_get(self, gpio):
         """
         Reads the value of gpio and returns "high" or "low"
         """
-        pass
+        raise NotImplementedError()
 
-    @abc.abstractmethod
     def gpio_set_high(self, gpio):
         """
         Sets a gpio high.
         """
-        pass
+        raise NotImplementedError()
 
-    @abc.abstractmethod
     def gpio_set_low(self, gpio):
         """
         Sets a gpio low.
         """
-        pass
+        raise NotImplementedError()
+
+    def info(self):
+        scr = self._usb.read_scr()
+        print("SCR: {}".format(scr.hex()))
+        cid = self._usb.read_cid()
+        print("CID: {}".format(cid.hex()))
+        csd = self._usb.read_csd()
+        print("CSD: {}".format(csd.hex()))
```

Comment:
This class is intended to be used as a library.
Please move the prints into the ``__main___.py`` (like with we do with the ``get`` or ``gpio get`` subcommands.